include/swift/AST/DiagnosticsSema.def

```diff
@@ -2446,6 +2446,9 @@ ERROR(no_MaxBuiltinIntegerType_found,none,
    "standard library error: _MaxBuiltinIntegerType is not properly defined", ())
 ERROR(no_MaxBuiltinFloatType_found,none,
    "standard library error: _MaxBuiltinFloatType is not properly defined", ())
+ERROR(integer_literal_overflows_maxwidth, none,
+   "integer literal needs %1 bits, exceeding limit of %0 bits",
+   (unsigned, unsigned, StringRef))
```

Comment:
Based on the use of this diagnostic the third parameter is supposed to be the number itself, but it's not used?